packages/workbox-build/src/lib/checkForDeprecatedOptions.js

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright 2018 Google LLC
```

Comment:
s/2018/2019/

(I'm not sure if we actually need to care about this?)

Reply:
I'll change it.